skyfield/elementslib.py

```diff
@@ -53,6 +53,8 @@ class OsculatingElements(object):
 
     """
     def __init__(self, position, velocity, time, mu_km_s):
+        if mu_km_s == 0: raise ValueError('`mu_km_s` is zero')
+        
```

Comment:
Could you put the `raise` on its own line ­— and maybe explain in the message why zero isn't a good value, since some callers might not know why? Thanks!

Reply:
Sure!